libkbfs/disk_block_cache_wrapped.go

```diff
@@ -46,40 +47,48 @@ func newDiskBlockCacheWrapped(config diskBlockCacheConfig, storageRoot string) (
 	if err != nil {
 		return nil, err
 	}
-	// TODO: enable sync cache in a subsequent PR.
-	var syncCache DiskBlockCache
-	//syncCacheRoot := filepath.Join(storageRoot, syncCacheFolderName)
-	//syncCache, err := newDiskBlockCacheStandard(config,
-	//	syncCacheLimitTrackerType, syncCacheRoot)
-	//if err != nil {
-	//	workingSetCache.Shutdown(context.Background())
-	//	return nil, err
-	//}
-	return &diskBlockCacheWrapped{
+	cache = &diskBlockCacheWrapped{
 		config:          config,
+		storageRoot:     storageRoot,
 		workingSetCache: workingSetCache,
-		syncCache:       syncCache,
-	}, nil
+		syncCache:       nil,
+	}
+	// TODO: enable sync cache in a subsequent PR.
+	// _ = cache.enableSyncCache()
+	return cache, nil
+}
+
+func (cache *diskBlockCacheWrapped) enableSyncCache() (err error) {
+	if cache.syncCache != nil {
+		return nil
+	}
+	syncCacheRoot := filepath.Join(cache.storageRoot, syncCacheFolderName)
+	cache.syncCache, err = newDiskBlockCacheStandard(cache.config,
```

Comment:
Seems like this would need locking, to prevent two LRU caches from being created for the same folder.